tests/tests_suite_test.go

```diff
@@ -171,6 +195,22 @@ func (s *existingSspStrategy) Init() {
 		return
 	}
 
+	appLabels := map[string]string{
+		common.AppKubernetesNameLabel:      "ssp-cr",
+		common.AppKubernetesManagedByLabel: "ssp-test-strategy",
+		common.AppKubernetesPartOfLabel:    "hyperconverged-cluster",
+		common.AppKubernetesVersionLabel:   "v0.0.0-test",
+		common.AppKubernetesComponentLabel: common.AppComponentSchedule.String(),
+	}
+	patch := buildLabelsPatchAdding(appLabels)
```

Comment:
It would be better if it is not necessary to patch the existing CR. The tests that modify the SSP CR revert it back to its original state in `AfterEach()`. That revert would remove these labels.

I have a few questions about this:
- Are these labels added by HCO in the current HCO version? If yes, we don't need to patch the SSP CR.
- If the labels are not present, what do we want to happen?
  -  Should the tests ignore the missing labels? The code in `labels.go` sets them to empty string.
  -  Should the tests fail? If yes, a good place for the check is in this function, to fail the whole test suite.

Reply:
The change is merged in hco, I don't know if that means they are already in a version.
`part-of` and `version` depend on them being set on the SSP CR and if they are not set there they are "allowed" but discouraged to be empty.
Technically the other labels are mandatory as the operator specifies them and if they are missing something is wrong.
But we could also drop them if we miss `part-of` and `version` in the form of "all or nothing".

Maybe @omeryahud has an opinion on this, I don't really mind either way. I think I'd prefer how it is and if `part-of` and `version` are missing on SSP we just keep them empty but still add the other labels.

Does that answer your question?

Reply:
Yes, thank you. In that case, this patch API call is not needed.

Reply:
Great, thanks. I'll remove it.